zkvm/src/types.rs

```diff
@@ -115,6 +116,26 @@ impl CommitmentWitness {
         let gens = PedersenGens::default();
         gens.commit(self.value.into(), self.blinding).compress()
     }
+
+    pub fn unblinded<T>(x: T) -> Self
```

Comment:
FYI: for short trait bounds you can put them inside angle brackets directly, if it looks better. E.g.:

```suggestion
    pub fn unblinded<T: Into<ScalarWitness>>(x: T) -> Self
```

In these short methods it may be easier to read.

Reply:
(that's not a nitpick worth addressing in this PR)